### PR TITLE
Adding Recording REST API

### PR DESCRIPTION
**Pull request by tomasgarzon**

### CODE REVIEW

File: zoomus/components/__init__.py [hunk 1]
```diff
@@ -8,4 +8,5 @@
     meeting,
     report,
     user,
-    webinar)
\ No newline at end of file
+    webinar,
+    recording)
```
Comment (prschmid) [inline zoomus/components/__init__.py]: Please put in alphabetical order.

File: zoomus/components/recording.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            kwargs['to'] = util.date_to_str(kwargs['to'])
+        return self.post_request("/recording/list", params=kwargs)
+
+    def delete(self, **kwargs):
+        util.require_keys(kwargs, ['meeting_id'])
+        return self.post_request("/recording/delete", params=kwargs)
+
+    def get(self, **kwargs):
+        util.require_keys(kwargs, ['meeting_id'])
+        return self.post_request("/recording/get", params=kwargs)
```
Comment (prschmid) [inline zoomus/components/recording.py]: Please write unit tests

File: zoomus/components/recording.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        util.require_keys(kwargs, 'host_id')
+        if kwargs.get('from'):
+            kwargs['from'] = util.date_to_str(kwargs['from'])
+        if kwargs.get('to'):
+            kwargs['to'] = util.date_to_str(kwargs['to'])
+        return self.post_request("/recording/list", params=kwargs)
+
+    def delete(self, **kwargs):
+        util.require_keys(kwargs, ['meeting_id'])
+        return self.post_request("/recording/delete", params=kwargs)
```
Comment (prschmid) [inline zoomus/components/recording.py]: Please write unit tests

File: zoomus/components/recording.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+class RecordingComponent(base.BaseComponent):
+    """Component dealing with all recording related matters"""
+
+    def list(self, **kwargs):
+        util.require_keys(kwargs, 'host_id')
+        if kwargs.get('from'):
+            kwargs['from'] = util.date_to_str(kwargs['from'])
+        if kwargs.get('to'):
+            kwargs['to'] = util.date_to_str(kwargs['to'])
+        return self.post_request("/recording/list", params=kwargs)
```
Comment (prschmid) [inline zoomus/components/recording.py]: Please write unit tests

**Comment by tomasgarzon** [comment]
@prschmid thanks for your feedback, sorry for the tests broken

**Comment by prschmid** [comment]
Thanks for your additions. Has been merged in to develop with commit f4bca761d5bc54d46ae25d5d74d85122b7a4b3da

**Pull request closed without merging**